<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="tests" name="Standalone Test(failed)">
    <classes>
      <class name="rahulsshettyacademy.StandaloneTest2">
        <methods>
          <include name="tearDown"/>
          <include name="launchApplication"/>
          <include name="StandaloneTest"/>
        </methods>
      </class> <!-- rahulsshettyacademy.StandaloneTest2 -->
    </classes>
  </test> <!-- Standalone Test(failed) -->
  <test thread-count="5" parallel="tests" name="Error Validation(failed)">
    <classes>
      <class name="rahulsshettyacademy.ErrorValidation">
        <methods>
          <include name="tearDown"/>
          <include name="launchApplication"/>
          <include name="LoginErrorValidationt"/>
        </methods>
      </class> <!-- rahulsshettyacademy.ErrorValidation -->
    </classes>
  </test> <!-- Error Validation(failed) -->
</suite> <!-- Failed suite [Suite] -->
